dmHelper/dmHelper.py

```diff
@@ -23,9 +26,26 @@ def __init__(self, bot):
         self.bot = bot
         self.message_queue : list = []
         self.priority_message_queue : list = []
+        self.errored_message_queue : list = [] # used to store DMs that were unable to be delivered
         self.actively_sending = False
+        self.bulk_role_manager_cog : BulkRoleManager = bot.get_cog("BulkRoleManager")
```

Comment:
Best to remove circular dependency. Adding roles is pretty simple, and I'd prefer to go with the role removal approach.